README.rst

```diff
@@ -37,13 +40,14 @@
 .. image:: https://img.shields.io/pypi/dm/invenio-previewer.svg
         :target: https://pypi.python.org/pypi/invenio-previewer
 
-.. image:: https://img.shields.io/github/license/inveniosoftware/invenio-previewer.svg
-        :target: https://github.com/inveniosoftware/invenio-previewer/blob/master/LICENSE
-
 
 Invenio module for previewing files.
 
-*This is an experimental developer preview release.*
+Features:
+
+- Allows previewing files
+- Bundled previewers out-of-the-box (PDF, ZIP, CSV, MARKDOWN, XML, JSON, PNG, JPG, GIF)
+- Custom build previewers
 
```

Comment:
What do you think about this?
```
- File previewing.
- Support for PDF, ZIP, CSV, MARKDOWN, XML, JSON, PNG, JPG and GIF out-of-the-box.
- Extensible API to create new previewers.
```

Or something similar, I based it on other packages such as [invenio-records](https://invenio-records.readthedocs.io/en/latest/).

Needs rebase also :)